### ISSUE TITLE
New namespaced map syntax in Clojure 1.9 alphas

### ISSUE THREAD

**Issue by bhagany** [sunng87/pyclj#16]
Clojure is adding a new shorthand syntax feature for maps with namespaced keys, and it uses this syntax for EDN as well.  For example:

```clojure
user> (pr-str {:foo/bar :foobar :foo/baz :foobaz})
"#:foo{:bar :foobar, :baz :foobaz}"
```

It would be very helpful for me to have this syntax supported, and I hope to send you a pull request for it soon.

**Comment by bhagany** [comment]
Here's a link to the JIRA ticket as well: http://dev.clojure.org/jira/browse/CLJ-1910

**Comment by sunng87** [comment]
Thank you @bhagany , this feature is implemented in your #17
